site/content/integrate/apps/_index.md

```diff
@@ -29,9 +29,20 @@ Apps are available as a Developers Preview and we're looking for your feedback!
 
 Read the [quick start guide]({{< ref  "quick-start-go" >}}) to learn how to write your first app.
 
-
 ## FAQ
 
+### When would you build an app vs a custom slash command vs a webhook vs a plugin?
+
+That depends on your use case, as they each have benefits. Apps can be advertised, shared, and installed by others via the Marketplace and users can get started with a slash command. Comparatively, a custom slash and webhooks command requires a lot of configuration, is very rigid - and is limited to your own server.
```

Comment:
> Apps can be advertised, shared, and installed by others via the Marketplace 

Not really right now. It's unclear how the Marketplace will allow installation for Apps besides the Mattermost build ones.

Reply:
Please also see https://github.com/mattermost/mattermost-developer-documentation/pull/804#discussion_r622019565 on the selling point of apps.